src/main/java/org/generationcp/middleware/dao/WorkbenchUserDAO.java

```diff
@@ -350,4 +357,60 @@ public WorkbenchUser getUserByFullName(final String fullname) {
 		}
 
 	}
+
+	public List<Integer> getActiveUserIDsByProjectId(final Long projectId, final String cropName) {
```

Comment:
As commented on `WorkbenchUser.GET_ACTIVE_USER_IDS_BY_PROJECT_ID`, it's better to rename this method to reflect we are only getting users with PROGRAM role types